kubedb/post_install.md

```diff
@@ -0,0 +1,27 @@
+## kubedb: A Kubernetes Operator for running MySQL, PostgreSQL, MongoDB, and other databases
+
+### Prerequisites
+
+Make sure a Longhorn is set as a default storage provider.  You can do so with the following command:
+
+```
+kubectl patch storageclass longhorn -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+```
```

Comment:
These instructions shouldn't be necessary now that https://github.com/civo/kubernetes-marketplace/commit/fa62fb2fac68670ff25abb6861b91294320fe725 is in the codebase. I would say remove them for clarity (if people have old clusters, they'll likely have come across this bug anyway or we can support them in Slack).

Reply:
Makes sense.  I'll remove that patch.